roles/confluent.test/molecule/rbac-mtls-rhel/verify.yml

```diff
@@ -168,6 +244,8 @@
         file_path: /etc/ksqldb/ksql-server.properties
         property: public.key.path
         expected_value: /var/ssl/private/public.pem
+    # TODO:  KSQL Currently still supports older TLS versions, and as of 6.0.0 does not have way to override to TLS 1.2 or later.
```

Comment:
setting `ssl.enabled.protocols` in the properties file did not work?

Reply:
Correct.  There is currently no way to override this.  They changed the webserver from jetty to vertx and omitted this in the last release.  Will ping you an internal thread.